File: Python/Code1.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for index1 in range(0, len(nums) - 1):
            num_left = target - nums[index1]
            nums_left = nums[index1 + 1:]
            try:
                return [index1, nums_left.index(num_left) + index1 + 1]
            except BaseException as error:
                continue
        return []
if __name__ == '__main__':
    reulst = Solution().twoSum([0,4,3,0], 0)
    pass

File: Code1.rb
# @param {Integer[]} nums
# @param {Integer} target
# @return {Integer[]}
def two_sum(nums, _target)
  nums.each_with_index do |_num, index|
    nums_left = nums[index + 1...:nums.length - 1]
    return [index, nums.find]
  end
end

result = two_sum([3, 3], 6)
puts result
